src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -236,19 +249,41 @@ def get_phylo_run(session, phylo_run_id):
     return phylo_run
 
 
+def resolve_filter_pango_lineages(
+    session: Session, template_args: Dict[str, Any]
+) -> Optional[List[str]]:
+    """Takes raw lineage filter, expands it. Helper for `resolve_template_args`
+
+    User can specify the lineages they want a tree build to filter on and are
+    able to use wildcards and some other, non-pango lineages as part of that.
+    Downstream needs Pango-only lineages. This handles using the originally
+    provided `filter_pango_lineages` arg and converting it. If arg was not
+    present in original template_args, returns None instead.
+    """
+    lineage_list = template_args.get("filter_pango_lineages")
+    if lineage_list is None:  # short-circuit if template arg was not present
+        return None
+    # Utility that does expansion depends on having set of all lineages.
+    all_lineages = {
+        query_result[0]  # query_result is 1-length tuple of just lineage
+        for query_result in session.query(PangoLineage.lineage).all()
```

Comment:
nit: Just a reminder that we should try to use SA 2.0 style queries (`sa.select()`) as much as possible

Reply:
Yeah, I waffled on that while coding this, but opted in favor of matching the format used by the rest of the file currently. If you think it's better to do 2.0 style even if it will be the odd duck out in the file, I'm happy to rewrite it.

Reply:
Your call. One less query to translate later would be nice, but it's not a reason to block this feature.

Reply:
Converted it over to v2 style